contents/handbook/people/team-structure/infrastructure.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Team Infrsstructure
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- James Greenhill (Team lead, Data/Infra Engineer)
+- Guido Iaquinti (Site Reliability Engineer)
+- Harry Wave (Full Stack Engineer)
```

Comment:
```suggestion
- Harry Waye (Full Stack Engineer)
```

Reply:
👋

Reply:
Haha, Sorry about that, I initially wrote it and laughed to myself and then forgot to correct it.